docs/book/src/user/troubleshooting.md

```diff
@@ -1,5 +1,64 @@
 # Troubleshooting
 
+## Troubleshooting Quick Start with Docker (CAPD)
+
+<aside class="note warning">
+
+<h1>Warning</h1>
+
+If you've run the Quick Start before ensure that you've [cleaned up](./quick-start.md#clean-up) all resources before trying it again. Check `docker ps` to ensure there are no running containers left before beginning the Quick Start.
+
+</aside>
+
+This guide assumes you've completed the [apply the workload cluster](./quick-start.md#apply-the-workload-cluster) section of the Quick Start using Docker.
+
+When running `clusterctl describe cluster capi-quickstart` to verify the created resources, we expect the output to be similar to this (**note: this is before installing the Calico CNI**).
+
+```shell
+NAME                                                           READY  SEVERITY  REASON                       SINCE  MESSAGE
+Cluster/capi-quickstart                                        True                                          46m
+├─ClusterInfrastructure - DockerCluster/capi-quickstart-94r9d  True                                          48m
+├─ControlPlane - KubeadmControlPlane/capi-quickstart-6487w     True                                          46m
+│ └─3 Machines...                                              True                                          47m    See capi-quickstart-6487w-d5lkp, capi-quickstart-6487w-mpmkq, ...
+└─Workers
+  └─MachineDeployment/capi-quickstart-md-0-d6dn6               False  Warning   WaitingForAvailableMachines  48m    Minimum availability requires 3 replicas, current 0 available
+    └─3 Machines...                                            True                                          47m    See capi-quickstart-md-0-d6dn6-584ff97cb7-kr7bj, capi-quickstart-md-0-d6dn6-584ff97cb7-s6cbf, ...
+```
+
+Machines should be started, but Workers are not ready due to Calico isn't installed yet. You should be able to see the containers running with `docker ps --all` and they should not be restarting.
```

Comment:
```suggestion
Machines should be started, but Workers are not because Calico isn't installed yet. You should be able to see the containers running with `docker ps --all` and they should not be restarting.
```